scripts/tools/silabs/README.md

```diff
@@ -0,0 +1,58 @@
+#FactoryDataProvider for EFR32 Matter device
+
+## Introduction
+
+The python FactoryDataProvider.py script allows a user to store a set of
+commissionable data in the matter nvm3 section of an EFR32 device. This script
+can handle writing to any MCU EFR32MG12 or EFR32MG24 family.
+
+# Prerequisites
+
+The users need to have in their hands a valid and matching set of spake2p+
+credentials - Setup Discriminator - Setup Passcode - Spake2p Iteration count -
+Spake2p Salt and one of the following: - Spake2p Verifier or the path to spake2p
+generator binary
+
+You can use this same spake2p generator to randomly generate a set of
+credentials or generate your a verifier based on the given Passcode, Iteration
+count and Salt.
+
+More info on the Spak2p generator and how to build it can be found here
+[Spake2p](https://github.com/project-chip/connectedhomeip/blob/master/src/tools/spake2p/README.md)
+
+# How to use FactoryDataProvider.py
+
+Using Python 3 call the 'FactoryDataProvider' script and provide the needed
+arguments. The script will validate the received data and if the device is
+connected to the host, write the data in the right nvm3 location.
+
+If the device is not connected, The script will generate an image that can be
+flashed to your device ulteriorly but you must provide the MCU family name.
+(EFR32MG12 or EFR32MG24).
+
+If you have more than one device connected to the host. Provide the J-tag serial
+number to ensure you are writing to the right device.
+
+| Arguments          | Conformace | Type    | Description                                                                             |
+| ------------------ | ---------- | ------- | --------------------------------------------------------------------------------------- |
+| --discriminator    | Required   | dec/hex | Provide BLE pairing discriminator.                                                      |
+| --passcode         | Required   | dec/hex | Provide the default PASE session passcode used to generate the spake2p verifier.        |
+| --spake2_iteration | Required   | dec/hex | Provide Spake2+ iteration count used to generate the spake2p verifier                   |
+| --spake2_salt      | Required   | string' | Provide Spake2+ salt used to generate the spake2p verifier.                             |
```

Comment:
Typo: quote marks in "optional" and "string" on this and the next two lines

Reply:
I was actually trying to simulate a subscript to indicate that there is a note for this after the table.

Usually it is number but i thought making it "prime" and "prime prime" would be readble. I'll find a better way